src/config/gobjects.rs

```diff
@@ -308,6 +307,22 @@ fn parse_object(
         .lookup("generate_builder")
         .and_then(Value::as_bool)
         .unwrap_or(false);
+    let init_function_expression = toml_object
+        .lookup("init_function_expression")
+        .and_then(Value::as_str)
+        .map(ToOwned::to_owned);
+    let clear_function_expression = toml_object
+        .lookup("clear_function_expression")
+        .and_then(Value::as_str)
+        .map(ToOwned::to_owned);
+
+    if (init_function_expression.is_some() && clear_function_expression.is_none())
+        || (init_function_expression.is_none() && clear_function_expression.is_some())
```

Comment:
Maybe `init_function_expression.is_some() != clear_function_expression.is_some()` ?

Reply:
I dislike comparing booleans with `==` and `!=` :)

Reply:
Ok, then I wait for CI and merge